src/components/Tooltip/index.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { Portal } from '../Portal'
+import { radius, color, space } from '../../theme'
+import { Global, css } from '@emotion/core'
+import { useTooltip, TooltipPopup } from '@reach/tooltip'
+
+const centered = (triggerRect, tooltipRect) => {
+  const triggerCenter = triggerRect.left + triggerRect.width / 2
+  const left = triggerCenter - tooltipRect.width / 2
+  const maxLeft = window.innerWidth - tooltipRect.width - 2
+  return {
+    left: Math.min(Math.max(2, left), maxLeft) + window.scrollX,
+    top: triggerRect.bottom + 8 + window.scrollY,
+  }
+}
+
+const Popup = styled(TooltipPopup)`
+  z-index: 10;
+  pointer-events: none;
+  position: absolute;
+  padding: ${space[8]} ${space[12]};
+  box-shadow: 2px 2px 10px hsla(0, 0%, 0%, 0.1);
+  white-space: nowrap;
+  font-size: inherit;
+  color: white;
+  border: 0;
+  border-radius: ${radius.md};
+  background: ${color.spaceMedium};
+`
+
+const Triangle = styled.div`
+  position: absolute;
+  z-index: 10;
+  width: 0;
+  height: 0;
+  border-left: 10px solid transparent;
+  border-right: 10px solid transparent;
+  border-bottom: 10px solid ${color.spaceMedium};
+`
+
+export function Tooltip({ children, label, 'aria-label': ariaLabel }) {
+  // get the props from useTooltip
+  const [trigger, tooltip] = useTooltip()
+  // destructure off what we need to position the triangle
+  const { isVisible, triggerRect } = tooltip
+  return (
+    <>
+      {React.cloneElement(children, trigger)}
+      {isVisible && (
+        // The Triangle. We position it relative to the trigger, not the popup
+        // so that collisions don't have a triangle pointing off to nowhere.
+        // Using a Portal may seem a little extreme, but we can keep the
+        // positioning logic simpler here instead of needing to consider
+        // the popup's position relative to the trigger and collisions
+        <Portal>
+          <Triangle
+            style={{
+              left:
+                triggerRect && triggerRect.left - 10 + triggerRect.width / 2,
```

Comment:
Same as other comment and maybe make a const out of this so we can explain why the `10` for example.